aiida/engine/utils.py

```diff
@@ -273,11 +272,8 @@ def set_process_state_change_timestamp(process: 'Process') -> None:
     description = PROCESS_STATE_CHANGE_DESCRIPTION.format(process_type)
     value = timezone.datetime_to_isoformat(timezone.now())
 
-    try:
-        manager = get_manager()
-        manager.get_backend_manager().get_settings_manager().set(key, value, description)
-    except UniquenessError as exception:
-        process.logger.debug(f'could not update the {key} setting because of a UniquenessError: {exception}')
+    backend = get_manager().get_backend()
+    backend.set_global(key, value, description)
```

Comment:
No to my knowledge. I didn't really understand why this was necessary, since it was, and still is an `UPSERT` operation (create or update existing), so there should never be an issue of uniqueness